playground/backend/cmd/server/controller.go

```diff
@@ -208,7 +208,7 @@ func setupCompileBuilder(lc *fs_tool.LifeCycle, sdk pb.Sdk, executorConfig *envi
 		WithCompiler()
 
 	switch sdk {
-	case pb.Sdk_SDK_JAVA:
+	case pb.Sdk_SDK_JAVA, pb.Sdk_SDK_GO:
```

Comment:
we could remove the `switch` statement. These actions should be done for each SDK, isn't it?